intelmq/bots/parsers/netlab_360/parser.py

```diff
@@ -46,6 +46,11 @@ def parse_line(self, line, report):
                 event.add('destination.port', value[2].replace('dport=', ''))
                 event.add('classification.type', 'scanner')
                 event.add('classification.identifier', 'mirai', overwrite=True)
+            elif report['feed.url'] in Netlab360ParserBot.HAJIME_SCANNER_FEED:
```

Comment:
Where is this variable defined?

Reply:
Just noticed that was missing and I have updated the file.